kernel-laws/shared/src/main/scala/cats/kernel/laws/discipline/BoundedTests.scala

```diff
@@ -47,12 +51,16 @@ object LowerBoundedTests {
 trait UpperBoundedTests[A] extends PartialOrderTests[A] {
   def laws: UpperBoundedLaws[A]
 
-  def upperBounded(implicit arbA: Arbitrary[A], arbF: Arbitrary[A => A], eqOA: Eq[Option[A]], eqA: Eq[A]): RuleSet =
+  def upperBounded(implicit arbA: Arbitrary[A], arbF: Arbitrary[A => A]): RuleSet =
     new DefaultRuleSet(
       "upperBounded",
       Some(partialOrder),
       "bound is greater than or equals" -> forAll(laws.boundGteqv _)
     )
+
+  @deprecated("use `upperBounded` without `Eq` paramerters", "2.12.1")
```

Comment:
```suggestion
  @deprecated("use `upperBounded` without `Eq` parameters", "2.12.1")
```

Reply:
Yeah, the evil of copy and paste here, thank you!